docs/historic.md

```diff
@@ -30,4 +30,24 @@ res.to_pandas().head()
 | 2022-06-01 07:00:00+00:00 |         13 |    62 |    12 |
 | 2022-06-01 07:30:00+00:00 |         13 |     0 |     0 |
 | 2022-06-01 08:00:00+00:00 |         12 |     0 |     0 |
-| 2022-06-01 08:30:00+00:00 |         12 |     0 |     0 |
\ No newline at end of file
+| 2022-06-01 08:30:00+00:00 |         12 |     0 |     0 |
+
+
+### Example of multi period request
+
+```python
+from solcast import historic
+import pandas as pd
+
+start_str = '2020-01-01'
+start = pd.Timestamp(start_str)
+days_required = 356
+days = 0
+
+while days < days_required:
+    startdate = start + pd.DateOffset(days=days)
+    startdate_str = startdate.strftime('%Y-%m-%d')
+    res = historic.radiation_and_weather(latitude=-33.856784, longitude=151.215297, start=startdate_str, duration='P31D',api_key='YOUR_API_KEY')
```

Comment:
This won't work as it will keep overwriting res.  What you need to do is define a list e.g.
`responses = []`, before the while loop.  Then do `responses.append(res)` after you get it.

That will leave you with a list of responses.  To get the data out concatenate it and clip to the required number of days, you will need something like the following.

```
data = []
for res in responses:
    if res.success:
        data.append(res.to_pandas())
    else:
        print(res.exception)
data = pd.concat(data)[:start+pd.Timedelta(days=days_required+1)]
```